riscv/src/continuations.rs

```diff
@@ -235,6 +235,12 @@ pub fn rust_continuations_dry_run<F: FieldElement>(
         .fold(None, |acc, (_, m)| acc.or(m.degree.clone()))
         .unwrap()
         .degree;
+
+    let length = match length {
+        Expression::Number(length, None) => length,
+        _ => unimplemented!(),
+    };
+
     let length = F::from(length).to_degree() as usize;
```

Comment:
Is this going through the field? I think it would be better to use `usize::try_from(length).unwrap()`